.github/workflows/main.yml

```diff
@@ -25,7 +25,12 @@ jobs:
         run: npm run lint
 
       - name: Test
-        run: npm run test -- --ci
+        run: npm run test -- --ci --updateSnapshot
```

Comment:
Won't updating the snapshots every time cause the test to never actually run?

Reply:
We are generating them right now, because we can't build them locally ...

Reply:
Will comment out in next commit

Reply:
Right, ok :+1: